e2e/framework/clients/rook_rest_client.go

```diff
@@ -0,0 +1,110 @@
+package clients
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/rook/rook/e2e/framework/enums"
+	"github.com/rook/rook/e2e/framework/utils"
+	"github.com/rook/rook/pkg/model"
+	rookc "github.com/rook/rook/pkg/rook/client"
+)
+
+type RookAPIClient struct {
+	rrc *rookc.RookNetworkRestClient
+}
+
+//Create Rook REST API client
+func CreateRookAPIClient(platform enums.RookPlatformType) *RookAPIClient {
+	var endpoint string
+	switch {
+	case platform == enums.Kubernetes:
+		rkh := utils.CreatK8sHelper()
+		//Start rook_api_external server via nodePort if not it not already running.
+		_, err := rkh.GetService("rook-api-external")
+		if err != nil {
+			rkh.ResourceOperation("create", "../../data/smoke/rook_api_external.yaml")
+		}
+		apiIp, err := rkh.GetPodHostId("rook-api", "rook")
+		if err != nil {
+			panic(fmt.Errorf("Host Ip for Rook-api service not found"))
+		}
+		endpoint = "http://" + apiIp + ":30002"
+	case platform == enums.StandAlone:
+		endpoint = "http://localhost:8124"
+	default:
+		panic(fmt.Errorf("platfrom type %s not yet supported", platform))
+	}
+
+	client := rookc.NewRookNetworkRestClient(endpoint, http.DefaultClient)
+
+	return &RookAPIClient{client}
+}
+
+func (a *RookAPIClient) URL() string {
```

Comment:
we just did it for consistency, since we have block, object and file wrappers, and those use rookClient if needed. 
Also this way users don't need to initiate rook rest api in between test, they already have at there disposal and use if they want too, also based on platform type, this wrapper picks the endpoint , e.g. users localhost for standalone,
or in k8s version figures out the hostIp of the service and exposes it on a nodePort